Formula/latexindent.rb

```diff
@@ -84,8 +84,8 @@ class Latexindent < Formula
   end
 
   resource "MRO::Compat" do
-    url "https://cpan.metacpan.org/authors/id/H/HA/HAARG/MRO-Compat-0.14_01.tar.gz"
-    sha256 "bc214d7964bc72f5a4015cc6b0d27376071cb64bd955280fea40c046b64d911a"
+    url "https://cpan.metacpan.org/authors/id/H/HA/HAARG/MRO-Compat-0.13.tar.gz"
```

Comment:
seems `0.13` is the latest stable version: https://metacpan.org/pod/MRO::Compat

Reply:
interesting, could that be the `0.14_01` not properly released?

Reply:
tag is in the github repo.

Reply:
cc @haarg (do you mind weighing in here?)

Reply:
The convention on CPAN is that versions with underscores indicate beta releases, so they aren't indexed.

I've just released 0.15, a new stable release.

Reply:
ah awesome, thanks for your help!

Reply:
Thanks haarg!